storage/rocksdb/ha_rocksdb.cc

```diff
@@ -417,6 +433,8 @@ static char *rocksdb_strict_collation_exceptions;
 static my_bool rocksdb_collect_sst_properties = 1;
 static my_bool rocksdb_force_flush_memtable_now_var = 0;
 static my_bool rocksdb_force_flush_memtable_and_lzero_now_var = 0;
+static my_bool rocksdb_enable_ttl = 1;
```

Comment:
Really? Interesting, then what is the point of a 'default' value for the GLOBAL SYSVAR macros. Seems to me it should be one or the other, explicit init of the static or explicit init via SYSVAR default, not some weird mix of both. Either way, I'll fix it but they have many my_bools just using 0|1 so I think rather than just fixing these in their cherry pick commits, I will leave them as merged and append a commit of my own to the whole change set that fixed them all in one shot and watch for more of the same in future merges.